flow_screen_components/lightningMapFSC/README.md

```diff
@@ -0,0 +1,34 @@
+# Lightning Map Component #
+
+Post describing this component:  https://unofficialsf.com/map-with-markers-in-flow/
+Original Aura Component off which the LWC is based:  https://unofficialsf.com/displaying-map-with-markers-in-flow-screen-using-lightning-components-by-terence-chiu/
+
+# lightningMapFSC
+Embed map in Flow passing in collection (of address attributes or geolocation), and allowing marker selection.
+
+### Summary:
+1)	Pass in Address or Geolocation and view on Map
+2)	Adjust Title of the Map
+3)	Show or hide the List View on the right side.
+
+### Admin Instructions:
+##### Enabling and Disabling Features in Flow:
+1)  Set Geolocation attribute to true if you plan to pass in Latitude and Longitude.  Otherwise you will pass in address.
+2)  For markerList (Address), you must pass in semi-colon delimited collection of strings in the following order:  City;Country;PostalCode;State;Street;Name;Icon;MarkerValue.
```

Comment:
I like to look for ways to create inputs that simplify the user's work. In this case, some options that occur to me: If I collect data in an upstream Screen element using the Address screen component, is there a way I could use that here without having to assemble the semicolon list? Perhaps have a bunch of additional text input fields?

Reply:
We could do that also (I believe the current mapFSC component you have in the collection does this).  In the other Aura component from blog post by Terence he went with the collection approach, and that's what I used as the launching point for the LWC update.  The reason is that, in thinking through use cases, I see many more instances where a collection of markers would be desired as opposed to a single marker based on an earlier entry.  Perhaps there is one that I'm missing though, or misunderstanding - highly likely!  

In doing this transposition, I could see a couple future enhancements that might be useful --
1)  Search for some address value(s), find matching records of some type and then display those on the map.
2)  Add filter/search logic on the same LWC screen to pare down results.